galera.spec

```diff
@@ -1,17 +1,15 @@
 Name:           galera
-Version:        25.3.5
-Release:        9%{?dist}
+Version:        25.3.12
```

Comment:
Hmm, this distgit was out of date, we actually have 25.3.16 in CBS and released in RDO common:
http://cbs.centos.org/koji/buildinfo?buildID=12268
@zzzeek are all fixes included there?